test/test_yahoofinancials.py

```diff
@@ -95,6 +95,13 @@ def test_yf_dividend_price(self):
 
     # Extra Module Methods Test
     def test_yf_module_methods(self):
+
+        # 10 Day Average Daily Volume
+        if isinstance(self.test_yf_stock_single.get_ten_day_avg_daily_volume(), int):
+            self.assertEqual(True, True)
+        else:
+            self.assertEqual(False, True)
```

Comment:
This can be rewritten like so:

`self.assertTrue(isinstance(self.test_yf_stock_single.get_ten_day_avg_daily_volume(), int))`

Reply:
Thank you for the feedback, I’ll include this in the next release.